content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -116,7 +116,11 @@ plot for the logged data automatically updated during the model training!
 
 ![](/img/dvclive-html.gif)
 
-> If you don't update the step number, the HTML report won't be generated.
+<admon type="info">
+ 
+If you don't update the step number, the HTML report won't be generated.
+
+</admon>
```

Comment:
Did you mean that text to be highlighted though? Block quotes can still be used for text we want to make less relevant (something you can skip, but without completely hiding it in a `<details>`).

Reply:
Q applies to every other non-emoji note turned into `<admon type="info">`.

Reply:
☝🏼 Up to you

Reply:
Yes, I wanted it to be highlighted although probably not taking as much space (#3381 )